lib/utils/__tests__/isStandardSyntaxKeyframesName.test.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const isStandardSyntaxKeyframesName = require('../isStandardSyntaxKeyframesName');
+
+describe('isStandardSyntaxKeyframesName', () => {
+	it('standard name', () => {
+		expect(isStandardSyntaxKeyframesName('slidein {}')).toBeTruthy();
```

Comment:
[suggest] `toBe(true)` / `toBe(false)` are more accurate than `toBeTruthy()` / `toBeFalsy()`. E.g.

```suggestion
		expect(isStandardSyntaxKeyframesName('slidein {}')).toBe(true);
```